esptfa_arima/arima_model/arima_model.py

```
@@ -22,69 +27,158 @@
 
     # make test number into int
-    test_data_long["test_number"] = test_data_long["test_number"].astype(int)
+    df["test_number"] = df["test_number"].astype(int)
+
+    # Sort by student and date for ARIMA processing
+    df = df.sort_values(["student_id", "date"])
+
+
+    # iterate over each student and transform into long format
+    features_list = []
+    for student_id, student_data in df.groupby("student_id"):
```

Ruff (0.14.10)

Loop control variable `student_id` not used within loop body

Rename unused `student_id` to `_student_id`

(B007)